guides/source/engines.md

```diff
@@ -207,7 +207,7 @@ instead be namespaced and called `Blorgh::Article`. In addition, the table for t
 model is namespaced, becoming `blorgh_articles`, rather than simply `articles`.
 Similar to the model namespacing, a controller called `ArticlesController` becomes
 `Blorgh::ArticlesController` and the views for that controller will not be at
-`app/views/articles`, but `app/views/blorgh/articles` instead. Mailers are namespaced
+`app/views/articles`, but `app/views/blorgh/articles` instead. Mailers and jobs are namespaced
```

Comment:
Should we mention helpers here too?

Reply:
Yeah, you're totally right, they are not mentioned before.